
syntax = "proto3";

package delivery.login;

service DeliveryLogin {
    rpc Login(LoginRequest) returns(LoginResponse);
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    string token = 1;
    enum Status {
        LOGGED = 0;
        INVALID = 1;
        ERROR = 2;
    }
    Status status = 2;
}
